Demo/Cql/Build/Cql.Targets.xml

```diff
@@ -9,11 +9,9 @@
 	</Target>
 	<Target Name="CQL to ELM"
 			DependsOnTargets="Download CQL to ELM CLI via Maven"
-			AfterTargets="Build"
-			Inputs="%(CQL.Identity)"

```

Comment:
No. It requires a clean to make any CQL file changes become Elm.

If the original intent was to process only the CQL files changes, it doesn't do that. I always have to do a clean and build to make a single CQL change become Elm, which of course processes all files. I am not sure how to register the CQL files with msbuild to do that selectively so I opted for a faster build all (and because I thought that was the design).

Without this targets change build time is about 2.5 minutes.
With this targets change build time is about 15 seconds.

Reply:
Oh well, then the intended speed up does not seem to work. Ok, let's go!

Reply:
Ok! So the intended speed-up does not seem to work. And 15 seconds is not worth spending too much time on...